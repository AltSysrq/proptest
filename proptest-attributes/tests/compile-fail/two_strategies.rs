//-
// Copyright 2019, 2020 The proptest developers
//
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.

use proptest_attributes::proptest;

#[proptest]
fn function_arn(
    #[strategy("[A-Z]{1}")]
    #[strategy("[a-z]{1}")]
    arg: String,
) {
}

fn main() {}
