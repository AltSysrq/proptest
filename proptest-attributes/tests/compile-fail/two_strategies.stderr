error: Expected at maximum one #[strategy] attribute
 --> $DIR/two_strategies.rs:6:5
  |
6 |     #[strategy("[a-z]{1}")]
  |     ^
